app/views/preferences_anything_else/edit.html.erb

```diff
@@ -0,0 +1,20 @@
+<% content_for :header_title, "What else should we know?" %>
```

Comment:
Not sure if Alan suggested nav text. Usually we've been using the same nav text for each page in a section (this is not 100% consistent but it is the case probably 80% of the time)

Reply:
Good idea! 9dc56f6